sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -127,3 +127,12 @@ export const INSTITUTION_LOCATION_NOT_VALID = "INSTITUTION_LOCATION_NOT_VALID";
  * Request for disability not allowed.
  */
 export const DISABILITY_REQUEST_NOT_ALLOWED = "DISABILITY_REQUEST_NOT_ALLOWED";
+/**
+ * Offering program year mismatch.
+ */
+export const OFFERING_PROGRAM_YEAR_MISMATCH = "OFFERING_PROGRAM_YEAR_MISMATCH";
+/**
+ * Offering does not belong to the location.
+ */
+export const OFFERING_DOES_NOT_BELONG_TO_LOCATION =
```

Comment:
Suggestion: on the same context `OFFERING_LOCATION_MISMATCH` ?

Reply:
IMO @dheepak-aot `MISMATCH` is when there is a direct comparison between two values. Let me know your thoughts devs